stories/Components/UIcomponents/Buttons/Cta_button/CtaButton.jsx

```diff
@@ -17,16 +17,18 @@ export const CtaButton = ({
     for_primary = 'without-arrow';
   }
 
+  const cls = (...classes) => (classes.filter(Boolean).length > 0) ? classes.filter(Boolean).join(' ') : null;
+
   return (
     <>
       {Type === 'Secondary'
         ? (
-          <a className={['button', `button-${type}`, `${state}`].join(' ')} role="button"  aria-pressed="false" href="#" >
+          <a className={cls('button', `button-${type}`, `${state}`)} role="button"  aria-pressed="false" href="#" >
             {label}
           </a>
         )
         : (
-          <a className={['button', `button-${type}`, ` button-${for_primary}`, `${state}`].join(' ')} role="button" aria-pressed="false" href="#">
+          <a className={cls('button', `button-${type}`, `button-${for_primary}`, `${state}`)} role="button" aria-pressed="false" href="#">
```

Comment:
@Shamimkhan321 We need `aria-pressed` only for toggle buttons and the state changes based on the toggle. This is not a toggle button but a CTA Button so please remove the aria-pressed attribute. Also on the above line.